packages/gulp-plugins/lib/tasks/unit-test.js

```diff
@@ -14,7 +14,8 @@ const configure = function configure (gulp, opts, env) {
       timeout: opts.testTimeout,
       traceWarnings: opts.test.traceWarnings,
       traceDeprecation: opts.test.traceWarnings,
-      color: true
+      color: true,
+      exit: Boolean(opts.test.exit)
```

Comment:
we usually use `!!opts.test.exit`

Reply:
for stuff like this we need ESLint rules... really, we shouldn't be talking about style in PRs.  it should all be automated.

when I was a new JS developer, I'd see `!!` and wonder what sort of clever trickery was going on.  I'd never seen `!!` before in any language.  so, that's my rationale for using `Boolean()` -- it should be more clear to newcomers to the language.

Reply:
I'm fine to standardize on either, I agree `Boolean` is more approachable. Without a lint rule, it will be hard to make any standardization changes however. Is there an eslint rule for this?

Reply:
if there isn't, it's pretty straightforward to make one